view/script/patient.js

```diff
@@ -105,4 +126,5 @@
         datesConfig.datasets[0].data = timeLeftData;
         return datesConfig;
     }
+         */
```

Comment:
The commented code will be used for daily chart in case we want a zoom type behavior.
As the weekly chart contains only 5-10 weeks data, we dont need zoom behavior for this one.